<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogRoutingInfoDialog</class>
 <widget class="QDialog" name="LogRoutingInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>561</width>
     <height>33</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Useful to identify object in a list.</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="enabledCheckBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Enabled</string>
   </property>
  </widget>
  <widget class="QLabel" name="regExpLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Reg Exp :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="regExpEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>561</width>
     <height>33</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Message must pass this to be displayed.</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="enableQDebugBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Send to qDebug</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="enableScreenCheckBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Send to Screen</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="enableFileCheckBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Send to file</string>
   </property>
  </widget>
  <widget class="QLabel" name="traceLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Trace Level:</string>
   </property>
  </widget>
  <widget class="QLabel" name="debugLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Debug Level:</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Info Level:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="traceSpinBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>64</width>
     <height>33</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="debugSpinBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>64</width>
     <height>33</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="infoSpinBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>64</width>
     <height>33</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="warnSpinBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>110</y>
     <width>64</width>
     <height>33</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="errorSpinBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>150</y>
     <width>64</width>
     <height>33</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="userSpinBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>190</y>
     <width>64</width>
     <height>33</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>160</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Error Level:</string>
   </property>
  </widget>
  <widget class="QLabel" name="warnLabel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Warn Level:</string>
   </property>
  </widget>
  <widget class="QLabel" name="userLabel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>200</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>User Level:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downCompButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Move the current filter down.</string>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upCompButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Move the current filter up.</string>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copyCompButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Move the current filter down.</string>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delCompButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Delete the selected filter.</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="component_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Components:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="insertCompButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Insert new filter at the currently selected filter.</string>
   </property>
   <property name="text">
    <string>Insert</string>
   </property>
  </widget>
  <widget class="QTableView" name="componentTableView">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>531</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="sampleTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>631</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="sampleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample Output:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LogRoutingInfoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LogRoutingInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
